### PR TITLE
Rework formatting and phrasing of README.md

### PR DESCRIPTION
**Pull request by trodery**
I reworked the formatting and phrasing of README.md. Grammatical and spelling errors were corrected. Any problems generated by the linter `markdownline` were corrected. I'm happy to make modifications any changes you may be unhappy with.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
+- `seek()`
+- `tell()`
 
 ## History
 
-This project is a fork of https://github.com/vinlyx/mgzip because we could not
-get in contact with vinlyx and we had bugfixes we wanted implemented. Thankyou
-to Vincent Li (@vinlyx) for all the hard work and if you ever come across this
-then feel free to get in contact.
+Created initially by Vincent Li (@vinlyx), this project is a fork of [https://github.com/vinlyx/mgzip](https://github.com/vinlyx/mgzip). We had several bug fixes to implement, but we could not him. The `pgzip` team would like to thank Vincent Li (@vinlyx) for his hard work. We hope that he will contact us when he discovers this project.
```
Comment (timhughes) [inline README.md]: Missing a word

but we could not **contact** him

I would also change all the 'he' and 'him' to 'they' and 'them'. You are probably correct to assume given the first name but you never know.

**Comment by timhughes** [comment]
Just one fix above and them I am happy to merge. Thanks for the contribution

**Review by timhughes — APPROVED** [review]
Fixed the minor changes I wanted to add

File: README.md (hunk 1 above)
Comment (trodery) [reply README.md]: I used Grammarly to check spelling and grammar and I'm quite surprised it missed that! Also, I don't contribute much on GitHub please pardon any silly mistakes I may make with pull requests, etc.

Comment (timhughes) [reply README.md]: No need to apologise.  This is the first package I have maintained publicly so I am learning as well.

**Pull request merged**